src/guiMainMenu.h

```diff
@@ -25,17 +25,19 @@ with this program; if not, write to the Free Software Foundation, Inc.,
 #include <string>
 #include <list>
 
-enum
-{
-	TAB_SINGLEPLAYER=0,
-	TAB_MULTIPLAYER,
-	TAB_ADVANCED,
-	TAB_SETTINGS,
-	TAB_CREDITS
+struct MainMenuDataForLua {
```

Comment:
Maybe it should be ScriptMainMenuData instead?  That removes the connotation that this structure is somehow reserved for Lua only.